casbin/casbin.go

```diff
@@ -109,7 +116,7 @@ func MiddlewareWithConfig(config Config) echo.MiddlewareFunc {
 // GetUserName gets the user name from the request.
 // Currently, only HTTP basic authentication is supported
 func (a *Config) GetUserName(c echo.Context) string {
-	username, _, _ := c.Request().BasicAuth()
+	username, _ := a.UserGetter(c)
```

Comment:
I'm not sure if ignoring the error it's OK. I know that our current implementation do that, but if this could be an opportunity to improve this code. Could you please change the signature of GetUserName to return also an error and check it on CheckPermission?